src/App.vue

```diff
@@ -103,7 +117,13 @@ import { EntityTypes, FilingCodes, NameRequestStates } from '@/enums'
     Actions,
     NameRequestInvalidErrorDialog,
     AccountAuthorizationDialog,
-    ConfirmDialog
+    ConfirmDialog,
+    AddPeopleAndRoles,
+    CreateShareStructure,
+    DefineCompany,
+    ReviewConfirm,
+    SignIn,
+    SignOut
```

Comment:
I have no idea how to specify "all components in views" here :/

If you have time left in this ticket (and/or the will to look it up), research this now, otherwise we'll cross this bridge later.

Reply:
I will do a little research on this as I am curious if it can be done as well.

Reply:
Updated.

Reply:
Of course! Brilliant!